docs/data/tree-view/simple-tree-view/customization/CustomAnimation.js

```diff
@@ -32,11 +29,6 @@ export default function CustomAnimation() {
     <SimpleTreeView
       aria-label="customized"
       defaultExpandedNodes={['1']}
-      defaultCollapseIcon={
```

Comment:
@noraleonte I removed the custom icon here to focus solely on the transition feature
Is that OK for you?

Reply:
Sure!